### ISSUE TITLE
Let notifications fire on service [up, down, or up & down]

### ISSUE THREAD

**Issue by fatflyingpigs** [louislam/uptime-kuma#508]
**Is it a duplicate question?**
Nope

**Is your feature request related to a problem? Please describe.**
Currently the notifications send on both service up and service down - but not only up, or only down.

**Describe the solution you'd like**
It would be nice if we could specify we only wanted the notification to send when the service went down (or when it comes back online) (or both).

**Describe alternatives you've considered**
-NA-

**Additional context**
I'm using this program to monitor (among other things) a minecraft server, and notify the server owner when his server is down. He's mentioned it's a bit annoying to get the 'server up' notification since he's literally the person who restarted the server.

**Comment by Genc** [comment]
This is no longer necessary. Can we close issue? @louislam

**Comment by CommanderStorm** [comment]
> This is no longer necessary. Can we close issue?

Why is this no longer necessary?

**Comment by Genc** [comment]
I thought the "upside down mode" feature met this request.
![image](https://github.com/louislam/uptime-kuma/assets/18603640/72899597-19e0-43ec-bbeb-112e5a629a03)

**Comment by CommanderStorm** [comment]
No, this FR is speciically to filter which event (`UP`/`DOWN`/`CERT-EXPIRY`/...) triggers alerts. Upside-down mode does not allow filtering
=> issue won't be closed ^^

**Comment by OneNerdyDad** [comment]
I would love to see this a possibility for webhooks. I have a few Alexa routines I can trigger via webhook but would need the ability to send a webhook based on up or down status vs trying to filter through another service first to filter that out.

Even if we could have a custom section for creating a trigger mechanism ourselves to tie in globally would be great. E.g. a custom_trigger.js where we can say if status is up do x, if down do y.

**Comment by mdl2934** [comment]
hello! i'm am working on a [pr](https://github.com/louislam/uptime-kuma/pull/7687) for this issue, and i need some help writing tests for my changes.
